src/comm.cc

```diff
@@ -1277,7 +1277,7 @@ commHandleWriteHelper(void * data)
         // and continue looking for a relevant one
     } while (clientInfo->hasQueue());
 
-    debugs(77,3, HERE << "emptied queue");
+    debugs(77,3, "emptied queue");
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.